wordpress-mobile/WordPress-iOS
# Reader post options: show message when following a site

Fixes: https://github.com/wordpress-mobile/WordPress-iOS/issues/15763
Ref: https://github.com/wordpress-mobile/WordPress-iOS/issues/15761

This shows a following message when a site is followed from a post card or post details. This is the same message that was previously displayed in post details (before I broke it with https://github.com/wordpress-mobile/WordPress-iOS/pull/15778).

~~There is not a notification for unfollowing. Stay tuned!~~ Unfollow notifications are implemented on https://github.com/wordpress-mobile/WordPress-iOS/pull/15786.

To test:
- In the Reader, show posts you are not following (ex: Discover, filter by topic).
- On a post card or in post details, tap the options menu button.
- Select `Follow Site`.
- Verify a toast message appears.

This functionality has not changed, but if you tap `Enable` on the notification, you should see `Success turn notifications on` in the logs as confirmation.

| ![menu](https://user-images.githubusercontent.com/1816888/106824396-f982be00-663f-11eb-8793-b18d398881f2.png) | ![message](https://user-images.githubusercontent.com/1816888/106824427-08697080-6640-11eb-8321-169fa2bfb423.png) |
|--------|-------|

PR submission checklist:
